src/components/SQFormScrollableCard/SQFormScrollableCard.js

```diff
@@ -123,11 +124,14 @@ function SQFormScrollableCard({
                 square={true}
                 className={classes.card}
               >
-                <CardHeader
-                  title={title}
-                  className={classes.cardHeader}
-                  titleTypographyProps={{variant: 'h4'}}
-                />
```

Comment:
Instead of a ternary here can you just use the `&&` to conditionally add the header
```
{!isHeaderDisabled && (
                  <CardHeader
                    title={title}
                    className={classes.cardHeader}
                    titleTypographyProps={{variant: 'h4'}}
                  />
                )}
```